server/catgenome/src/main/java/com/epam/catgenome/manager/gene/GffManager.java

```diff
@@ -1039,7 +1039,7 @@ private void fillMinMax(final DimStructure structure, final List<Alignment> alig
                 }
             }
             return chainMinMax;
-        }).collect(Collectors.toMap(ChainMinMax::getName, cMM -> cMM));
+        }).collect(Collectors.toMap(ChainMinMax::getName, Function.identity()));
```

Comment:
I would prefer to not to change any code if it doesn't brings new functionality or improvements. It is obvious that this two lines are functionally identical and in this case we can change it to make it beauty, but just for the note, if it is possible - let's leave such parts as is. It will help us to reduce number of changes to check, reduce possibility to add some "occasional" bugs and also will allow us to concentrate on real changes

Reply:
Fixed and understood!